src/coreclr/jit/gentree.h

```diff
@@ -6624,7 +6624,28 @@ struct GenTreeHWIntrinsic : public GenTreeJitIntrinsic
 
     static bool Equals(GenTreeHWIntrinsic* op1, GenTreeHWIntrinsic* op2);
 
-    genTreeOps HWOperGet(bool* isScalar) const;
+    static NamedIntrinsic HWIdGetForUnOp(
+        Compiler* comp, genTreeOps oper, GenTree* op1, var_types simdBaseType, unsigned simdSize, bool isScalar);
+    static NamedIntrinsic HWIdGetForBinOp(Compiler*  comp,
```

Comment:
The name seems little odd. Can it be GetHWIdForUnOp, GetHWIdForCmpOp?

Reply:
It could be, I'm mostly just matching the name of `HWOperGet`.

If I were to fix this one it'd likely be `GetHWIntrinsicIdForBinOp` and then changing `HWOperGet` to be `GetOperforHWIntrinsicId`.

Reply:
Went ahead and updated this to use the names above.